client/services/AimService.tsx

```diff
@@ -102,7 +102,14 @@ export const getAim = async (id: string) => {
 
 export const getAims = async () => {
   try {
-    const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL_SSR}/aims`)
+    return [
```

Comment:
Why are we hardcoding this?

Reply:
we still need to figure out how to get a token or id into this function, returned hardcoded aim bc that isn't done yet. I removed the hardcoded return, but the original problem still exists.